fedimint-core/src/util/mod.rs

```diff
@@ -217,6 +218,81 @@ pub async fn write_new_async<P: AsRef<Path>, C: AsRef<[u8]>>(
         .await
 }
 
+#[derive(Debug, Clone)]
+pub struct Spanned<T> {
```

Comment:
Comment?

Reply:
I'm confused what is this trying to accomplish.

Reply:
i don't want to this backport from master.

it is utility struct to enter a span whenever you want to use the wrapped object

Reply:
Missed opportunity to call it "spanner". :D